htdocs/edittags.bml

```diff
@@ -103,8 +103,11 @@ body<=
     $ret .= "<script type='text/javascript'> var cur_taglist = '$logtagstr'; </script>";
 
     $ret .= '<table summary="" class="edittbl" cellpadding="0" cellspacing="0" width="50%">';
+    if ( $u->is_community ) {
+        $ret .= "<tr><td class='l'>$ML{'.poster'}</td><td>" . $ent->poster->ljuser_display . "</td></tr>"
+        };
     $ret .= "<tr><td class='l'>$ML{'.subject'}</td><td>$subj</td></tr>" if $subj;
-
+    
```

Comment:
Pretty sure that's actually git getting confused about whitespace moving? i.e. there was already a line of whitespace at this location, it's just that git thinks I've removed it and replaced it with a new one. If you want the line of whitespace gone altogether that's cool and I'll do it, but can I have clarification please? :-)